client/src/constants.js

```diff
@@ -1 +1 @@
-export const BASE_URL = "http://localhost:8000/api/";
+export const BASE_URL = "http://localhost:8000/api";
```

Comment:
This change will affect some pages in the DashboardLayout folder and Add/ Edit product form. Please make the relevant changes in the necessary files.